fe/fe-core/src/main/java/org/apache/doris/qe/MysqlConnectProcessor.java

```diff
@@ -124,7 +105,7 @@ private void handleExecute() {
                     for (int i = 0; i < paramCount; ++i) {
                         int typeCode = packetBuf.getChar();
                         LOG.debug("code {}", typeCode);
-                        prepareCtx.stmt.placeholders().get(i).setTypeCode(typeCode);
+                        prepareStmt.placeholders().get(i).setTypeCode(typeCode);
```

Comment:
in nerieds all these info should be record into context mapping by PlaceHolderId, not PlaceHolder itself

Reply:
done